ReactSkia/core_modules/RSkImageLoader.cpp

```diff
@@ -117,6 +110,40 @@ void RSkImageLoaderModule::getImageSize(std::string uri, CxxModule::Callback res
     RNS_LOG_DEBUG("Send Request to network");
     sharedCurlNetworking->sendRequest(remoteCurlRequest,folly::dynamic::object());
     imageRequestList_.insert(std::pair<std::string, std::shared_ptr<CurlRequest> >(uri,remoteCurlRequest));
+  }else if(std::filesystem::exists(path)|| (uri.substr(0, 14) == "file://assets/")){ // Check if ION PATH exits.  
```

Comment:
comment must be Check if assets path exists. Also, can we have a telephonic review on this part ?